app/src/main/java/com/jerboa/MainActivity.kt

```diff
@@ -165,17 +165,14 @@ class MainActivity : AppCompatActivity() {
                     ) {
                         LoginActivity(
                             navController = navController,
-                            accountViewModel = accountViewModel,
```

Comment:
I saw in the workshop, the viewModel parameters weren't passed in directly, they used the factory in the composable instead, like https://github.com/dessalines/jerboa/pull/973/files#diff-6ca2c69ae054a88358d16c98f47efd02bb16eae48a09295b4db9bd5e99ba8e76R25.